<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1622</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pilot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="3">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="ambient_temp" native="true">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>250</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="Background2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>150</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <widget class="QFrame" name="Thermo2">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>105</y>
                <width>40</width>
                <height>40</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 255);
border: 2px solid;
border-radius: 20px;
border-color: rgb(255, 255, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QLabel" name="temp_value">
               <property name="geometry">
                <rect>
                 <x>5</x>
                 <y>5</y>
                 <width>30</width>
                 <height>28</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
               </property>
               <property name="text">
                <string>19Â°</string>
               </property>
              </widget>
             </widget>
             <widget class="QFrame" name="Thermo">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>10</y>
                <width>20</width>
                <height>101</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(14, 14, 255, 255), stop:1 rgba(255, 0, 0, 255));
border-top-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-bottom-color: rgb(0, 0, 255);
border-radius: 10px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <widget class="QProgressBar" name="temp_bar">
               <property name="geometry">
                <rect>
                 <x>7</x>
                 <y>8</y>
                 <width>5</width>
                 <height>135</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">border: none;
border-radius: 2px;
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="value">
                <number>77</number>
               </property>
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="format">
                <string/>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="pressure" native="true">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="Background">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QFrame" name="RPB_BACK">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 255);
border:none;
border-radius: 59px;</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Shape::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Shadow::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QFrame" name="RPB_PATH">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.823928 rgba(85, 255, 255, 255), stop:0.828442 rgba(0, 0, 124, 255));
</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QFrame" name="RPB_Front">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>4</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: rgb(0, 0, 127);
border: none;
border-radius:50;</string>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::Shape::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Shadow::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="rpb_kpa">
                         <property name="maximumSize">
                          <size>
                           <width>100</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>12</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                         </property>
                         <property name="text">
                          <string>0 kPa</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignmentFlag::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="rpb_perc">
                         <property name="maximumSize">
                          <size>
                           <width>100</width>
                           <height>40</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>12</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                         </property>
                         <property name="text">
                          <string>0%</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignmentFlag::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2" colspan="7">
      <widget class="QLabel" name="MainCameraView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>704</width>
         <height>396</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/HYDROROBOTICSLOGO.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="3">
      <widget class="QLabel" name="SecondaryCameraView2">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>400</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/HYDROROBOTICSLOGO.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="SecondaryCameraView1">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100000</width>
         <height>400</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/HYDROROBOTICSLOGO.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
